docs/quantum_research/index.rst

```diff
@@ -0,0 +1,64 @@
+.. _index_quantum_research:
+
+================
+Quantum Research
+================
+
+.. toctree::
+    :hidden:
+    :maxdepth: 1
+
+    index_get_started
+    index_about
+    index_using
+
+.. sections-start-marker
+
+The :ref:`index_quantum_research` section shows how to use |dwave_short_tm|
```

Comment:
Minor, but I find it annoying when links link to the page you're currently on.
```suggestion
This section shows how to use |dwave_short_tm|
```

Reply:
I agree but in this case I'm more worried that someone just scanning this front page will mistake "This section shows" as referring to the entire page/site.